addons/survey/views/survey_templates_statistics.xml

```diff
@@ -43,35 +34,32 @@
             <t t-call="survey.user_input_session_leaderboard"/>
         </div>
         <div t-if="survey.scoring_type in ['scoring_with_answers', 'scoring_without_answers']">
-            <h2 class="mt16 text-uppercase d-none">Results Overview</h2>
             <hr class="my-2"/>
             <div class="row mt-2">
-                <div class="col">
-                    <h2 class="text-primary fs-4" t-out="survey.question_count"/>
-                    <h4 class="fs-6">Questions</h4> 
+                <div class="col-4 col-sm-2">
+                    <h4 class="text-primary" t-out="survey.question_count"/>
+                    <h6>Questions</h6>
                 </div>
-                <div class="col">
-                    <h2 class="text-primary fs-4" t-out="survey.answer_count"/>
-                    <h4 class="fs-6">Registered</h4> 
+                <div class="col-4 col-sm-2">
+                    <h4 class="text-primary" t-out="survey.answer_count"/>
+                    <h6>Registered</h6>
                 </div>
-                <div class="col">
-                    <h2 class="text-primary fs-4" t-out="survey.answer_done_count"/>
-                    <h4 class="fs-6">Completed</h4> 
+                <div class="col-4 col-sm-2">
+                    <h4 class="text-primary" t-out="survey.answer_done_count"/>
+                    <h6>Completed</h6>
                 </div>
-            </div>
-           <div class="row mt-2">
-                <div class="col-4">
-                    <h2 class="text-primary fs-4"><t t-call="survey.survey_remove_unnecessary_decimals">
-                        <t t-set="value_to_format" t-value="survey_data['global_success_rate']"/></t>%</h2>
-                    <h4 class="fs-6">Success rate</h4>  
+                <div class="col-4 col-sm-2">
+                    <h4 class="text-primary"><t t-call="survey.survey_remove_unnecessary_decimals">
+                        <t t-set="value_to_format" t-value="survey_data['global_success_rate']"/></t>%</h4>
+                    <h6>Success rate</h6>
                 </div>
-                <div t-if="survey.answer_duration_avg" class="col-4">
-                    <h2 class="text-primary fs-4" t-out="survey.answer_duration_avg" t-options="{'widget': 'float_time'}"/>
-                    <h4 class="fs-6">Average Duration</h4> 
+                <div t-if="survey.answer_duration_avg" class="col-4 col-sm-2">
+                    <h4 class="text-primary" t-out="survey.answer_duration_avg" t-options="{'widget': 'float_time'}"/>
+                    <h6>Average Duration</h6>
                 </div>
-                <div t-if="survey.answer_score_avg" class="col-4">
-                    <h2 class="text-primary fs-4"><t t-out="round(survey.answer_score_avg, 2)"/>%</h2>
-                    <h4 class="fs-6">Average Score</h4> 
+                <div t-if="survey.answer_score_avg" class="col-4 col-sm-2">
```

Comment:
If I count correctly we have 6 times a col-4 ? Should bee 12-columns based right ?

On the other hand we have 6 time "col-sm-2", which makes more sense, shouldn't it always be "col-2" then?

Or does the overflow go into another row, which is what we want for smaller devices maybe?

Reply:
Isn't this specifically for the case where you don't want to specify the column size though?

I mean we could still implement it, just specify "col" instead of "col-4 col-sm2" and keep your w-100 div for sm sizes and larger, would slightly simplify, what do you think?

Reply:
Hello @awa-odoo :slightly_smiling_face: 
It works! Updated! :smile: 
Thanks!